f600233d1b154d507b782f84604bcb12/update/sys_ws_operation_1946bd8edb64a918b1227ea5f396197f.xml

```diff
@@ -10,7 +10,17 @@
         <operation_script><![CDATA[(function process(/*RESTAPIRequest*/ request, /*RESTAPIResponse*/ response) {
 	try {
 		const eda = new Eda();
-		return eda.getTaskOrderData(request.queryParams.taskOrderNumber,request.queryParams.acqusitionPackageSysId);			
+		const toDetails = eda.getTaskOrderData(request.queryParams.taskOrderNumber);
+		if (toDetails) {
```

Comment:
Aren't we always going to find something here if the TO is valid? That is, won't `toDetails` be truthy even if we don't have an existing TO for it?